src/components/bookmarklet/BookmarkletStep2.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <BackToAssetLibrary anchor="asset-library" :disabled="isLoading" />
+    <div class="align-center d-flex">
+      <div class="pr-3">
+        <font-awesome-icon icon="bookmark" size="lg" />
+      </div>
+      <div>
+        <h2>{{ pageTitle }}</h2>
+      </div>
+    </div>
+    <div v-if="!isLoading" class="pt-2 w-100">
+      <img
+        :aria-label="`Screenshot showing ${pageTitle}`"
+        :alt="`Screenshot showing ${pageTitle}`"
+        :src="screenshot"
+      />
+      <div>
+        First, enable your browser's <b>TODO: data-ng-bind="toolbar"</b>.
```

Comment:
@johncrossman Is this TODO here intentionally?

Reply:
@lyttam  Yes it is. It'll be dealt with in next PR. :)